app/src/main/res/layout/activity_wallet_details.xml

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We probably have to wrap the layout in a scrollview, otherwise on smaller screens it leads to overlaps. 
This is from a Nexus One.
![Screenshot_1585387361](https://user-images.githubusercontent.com/11649986/77820001-6273de00-70df-11ea-89d9-e010ca49cabf.png)